### ISSUE TITLE
Questions about code details in Video-R1

### ISSUE THREAD

**Issue by pitlover** [tulerfeng/Video-R1#90]
Hi, thanks for the great work on Video-R1!
While studying the training and evaluation code, I came up with some questions.

**1. Is it not possible to perform **evaluation** during the **SFT and RFT training stages**?**
When loading the dataset using:
`Dataset = DatasetDict({"train": Dataset.from_json(script_args.dataset_name)})`
it seems there is no dataset["test"] split.
Are the datasets (e.g., Video-R1-260k.json, Video-R1-COT-165k.json) intended to be train-only, or should we manually create a validation split for evaluation?

**2. Hard-coded dataset fallback risk**
I found the following code in the Video-R1/src/r1-v/src/open_r1/trainer/grpo_trainer.py file.
`try:
      image_inputs, video_inputs, video_kwargs = process_vision_info(input_copy, return_video_kwargs=True)
  except Exception as e:
      print(f"process_vision_info error, using fixed data, {e}") # TODO ???? 
      if inputs[0]['data_type'] == 'image':
          input_copy[0]['content'][0]['image'] = os.getcwd() + "/Video-R1-data" + '/Math/Multimath-300k/17ff4c7d14c388134de02381b1fc2824.png'
      elif inputs[0]['data_type'] == 'video':
          input_copy[0]['content'][0]['video'] = os.getcwd() + "/Video-R1-data" + '/LLaVA-Video-178K/liwei_youtube_videos/videos/youtube_video_2024/ytb_7nRmsEw7nsE.mp4'`

If dataset loading fails, the code appears to replace it with a specific hard-coded dataset.
Does this fallback mechanism affect the actual training results or introduce inconsistency between experiments?

**3. Meaning of the hard-coded 0.8 threshold**
In the same file (grpo_trainer.py ), there’s a line:
`if acc_mean >= 0.8 * shuffled_acc_mean: ### Here 
    mask = temporal_rewards_per_func[:, 0] > 0.1
    temporal_rewards_per_func[mask, 0] = temporal_rewards_per_func[mask, 0] + 0.3
    temporal_rewards = torch.tensor([1.0]).to('cuda')`

What does the constant 0.8 represent here?
It doesn’t seem to appear in Equation (1) of the paper, so I’d like to understand the rationale behind this threshold.

**4. Difference between 'question' and 'problem' keys in evaluation JSONs**
In the evaluation dataset JSON (ex. eval_mvbench.json), the values for the keys question and problem are always identical.
Is this intentional? If so, what’s the conceptual difference between these two fields?

Thanks in advance for your clarification!
I think these details will help a lot of users better understand the training and evaluation pipeline.
